src/plugin-manager.js

```diff
@@ -8,7 +8,7 @@ import {FrameworkConfiguration} from 'aurelia-framework';
  * @export
  * @interface PluginDefinition
  */
-export interface PluginDefinition {
+interface PluginDefinition {
```

Comment:
It's not the same as a default export, is it? Or does typescript export all interfaces anyway?